src/server/array_func.c

```diff
@@ -716,6 +980,19 @@ setup_arrayjob_attrs(attribute *pattr, void *pobj, int mode)
 		if ((pjob->ji_ajtrk = mk_subjob_index_tbl(pjob->ji_wattr[(int)JOB_ATR_array_indices_submitted].at_val.at_str,
 			                                      JOB_STATE_QUEUED, &pbs_error, mode)) == NULL)
 			return pbs_error;
+		
+		prange = pjob->ji_wattr[(int)JOB_ATR_array_indices_submitted].at_val.at_str;
+		if (!prange)
```

Comment:
there is no need to check for not NULL, it is NULL will never reach this point as it will crash server in mk_subjob_index_tbl()->parse_subjob_index()->`while (isspace((int) *pc)` + setup_arrayjob_attrs is action func of ATTR_array_indices_submitted, means it will be called only when value is set from req_queuejob()